src/Config.js

```diff
@@ -165,12 +166,21 @@ module.exports = class extends ChainedMap {
       );
     }
 
+    if (!omit.includes('node') && 'node' in obj) {
+      if (typeof obj.node === 'object') {
```

Comment:
`true` for `obj.node == []`. Though the input is unexpected it might cause issues.